.pylintrc

```diff
@@ -7,7 +7,7 @@ extension-pkg-whitelist=
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=
+ignore=orcaparser.py,logfileparser.py,data.py,ccio.py,utils.py
```

Comment:
Can you specify the whole directory instead of listing individual files? This would I guess also catch our own `utils.py` file if we had it?

Reply:
I tried the `ignore-patterns` but it seems buggy to me as it was not properly picking up those files. It's okay as there is no other `utils.py` file in the plugin.

Reply:
Also, these files are not gonna sit in the repository for long time.